src/app/EventManagement.cpp

```diff
@@ -747,6 +763,13 @@ CHIP_ERROR EventManagement::FetchEventParameters(const TLVReader & aReader, size
         envelope->mCurrentTime.mType  = Timestamp::Type::kEpoch;
         envelope->mCurrentTime.mValue = epochTime;
     }
+
+    if (TLV::IsProfileTag(reader.GetTag()) && reader.GetTag() == TLV::ProfileTag(kEventManagementProfile, kFabricIndexTag))
+    {
+        FabricIndex fabricIndex = 0;
```

Comment:
Please use kUndefinedFabricIndex instead of 0

Reply:
Done, thanks, It is ready for formal review.